api/v1alpha1/tracing_types.go

```diff
@@ -114,3 +116,18 @@ type RequestHeaderCustomTag struct {
 	// +optional
 	DefaultValue *string `json:"defaultValue,omitempty"`
 }
+
+// TracingProviderSettings defines optional configuration for tracing providers.
+type TracingProviderSettings struct {
```

Comment:
IIRC, these configurations are zipkin only, so it should under TracingProvider like following:
```
provider:
  type: Zipkin
  zipkin:
     traceId128Bit:
     sharedSpanContext:
```

Reply:
Agree - I have moved things around.